### ISSUE TITLE
TrainReservation & TaxiReservation: 'label' has an empty character at beginning of string.

### ISSUE THREAD

**Issue by JukoJones** [schemaorg/schemaorg#834]
While sorting the data by "label" I noticed that the order was showing up incorrectly. Upon Further investigation I noticed that both 'TaxiReservation' & 'TrainReservation' labels are proceeded by an empty character in the `schema.nt`, `schame.rdfs` files of version: 2.1. This was not present in the `schema-all.html` version. I did not check previous versions, or other filetypes.

https://github.com/schemaorg/schemaorg/tree/sdo-phobos/data/releases/2.1

**Example:**
Notice that the string value for each "label" is proceeded by an empty character.
``` Javascript
    {
      "id": "TaxiReservation",
      "type": "Class",
      "comment": "A reservation for a taxi.Note: This type is for information about actual reservations, e.g. in confirmation emails or HTML pages with individual confirmations of reservations. For offers of tickets, use http://schema.org/Offer.",
      "label": " TaxiReservation",
      "subClassOf": {
        "id": "Reservation"
      }
    },
    {
      "id": "TrainReservation",
      "type": "Class",
      "comment": "A reservation for train travel.Note: This type is for information about actual reservations, e.g. in confirmation emails or HTML pages with individual confirmations of reservations. For offers of tickets, use http://schema.org/Offer.",
      "label": " TrainReservation",
      "subClassOf": {
        "id": "Reservation"
      }
    }
```

**Comment by danbri** [comment]
Thanks. Noted as a bug.

**Comment by danbri** [comment]
Fixed from v3.1, thanks.
